go/interpreter/lfvm/gas.go

```diff
@@ -329,7 +319,7 @@ func gasSStoreEIP2200(c *context) (tosca.Gas, error) {
 	// If we fail the minimum gas availability invariant, fail (0)
 	if c.gas <= SstoreSentryGasEIP2200 {
```

Comment:
This check is done again in line 381. I don't see at this moment where is the gas modified in between. 
- What does the coverage say about these two ifs (320 and 381)?
- Would it be equivalent to have only one of the two?

Reply:
they are in 2 different functions, `gasSStoreEIP2200(c *context)` ends in line 371, `gasSStoreEIP2929(c *context)` starts in line 373